assets/src/components/SelectCard.js

```diff
@@ -117,23 +136,54 @@ const SelectCard = props => {
 
   function getCardImage (cardData) {
     if (cardData && cardData.image) {
-      return <CardMedia className={classes.media} image={cardData.image} title={cardData.title} />
+      return (
+        <>
+          <Link className={classes.viewLink} tabIndex={-1} to={cardData.path}>
+            <CardMedia className={classes.media} image={cardData.image} title={cardData.title} />
+          </Link>
+        </>
+      )
     } else {
       return null
     }
   }
 
+  function getHelpLink (cardData) {
+    if (cardData.helpUrl) {
+      return (
+        <Typography gutterBottom variant='h5' component='h4' className={classes.title}>
+          <Tooltip title={'About ' + cardData.title}>
+            <MUILink className={classes.infoLink} href={cardData.helpUrl} target='_blank' rel='noopener noreferrer'>
+              <InfoIcon />
+            </MUILink>
+          </Tooltip>
+        </Typography>
+      )
+    }
+  }
+
   function getLinkContents (cardData) {
     const cardImage = getCardImage(cardData)
 
     const cardContent = (
       <CardContent className={classes.content}>
-        <Typography gutterBottom variant='h5' component='h4' className={classes.title}>
-          {cardData.title}
-        </Typography>
-        <Typography component='p' className={classes.description}>
-          {cardData.description}
-        </Typography>
+        <Grid container className={classes.titleArea}>
+          <Grid item xs={11}>
+            <Link tabIndex={-1} to={cardData.path} className={classes.viewLink}>
+              <Typography gutterBottom variant='h5' component='h4' className={classes.title}>
```

Comment:
Nit-pick: We have a variant mapping in `globals.js` for Web accessibility purposes. I think the `h5` variant looks good, but I think we actually want it to show up as an `h2` (or maybe `h1`). I think `h5` maps to `h1`, so maybe just change `component` here to `h2`. This is mostly to support screen reading, I think; we want to avoid skipping header levels.

Reply:
So it has been using h4 all along, and changing that makes the text larger.. is there some functionality here that makes the screen reader treat it as a h2 but the browser render it as a h4?  Or are you saying we should just make it a h2?

Reply:
We want the style to be `h5` (I think) and the actual HTML element to be `h2`. Based on what I'm seeing in the [`Typography` API](https://material-ui.com/api/typography/), you can change this by modifying `component` to be `h2`. You're saying that affected the style? My understanding is `variant` controls the style, but I could be wrong.

Reply:
Ok I changed the h4 to h2.